pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.wdl

```diff
@@ -53,6 +54,12 @@ workflow WholeGenomeGermlineSingleSample {
 
     Boolean provide_bam_output = false
     Boolean use_gatk3_haplotype_caller = true
+    Boolean run_dragen_mode = false
```

Comment:
Yes (even though you can't use dragen mode with GATK3 obviously). In order to be functionally equivalent, users must set use_gatk3_haplotype_caller to false, run_dragen_mode to true, use_bwa_mem to false, use_spanning_event_genoytping to false and perform_bqsr to false. They might want to enable/disable features individually though if they're not interested in functional equivalence. It might make sense to add an additional single argument that automatically overrides all these options if people care about functional equivalence. If you think that's a good idea feel free to add that!

Reply:
we might want to add a check to ensure you don't run dragen mode with GATK3, otherwise I think we can just document the different options available

Reply:
@michaelgatzen could you include with this PR an example inputs json that will run the pipeline in DRAGEN mode with all the correct parameters set?

Reply:
@kbergin The user education team is working on a comprehensive documentation for these options, but sure, I can provide an inputs json with the options. Where do you want me to put it, and which input data should it use?

Reply:
Let's make a dragen fe and maximum performance version of this example inputs. It can stay in the same folder 
https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/dna_seq/germline/single_sample/wgs/input_files/WholeGenomeGermlineSingleSample.inputs.plumbing.json

Reply:
Done, but not resolving until the reference files have been moved to Google's bucket and we verify that the example inputs are working